pages/api/profiles.ts

```diff
@@ -0,0 +1,28 @@
+import { PrismaClient } from '@prisma/client'
```

Comment:
**issue:** Next 13 introduced `app` directory instead of `pages`. We will have more possibilities when it comes to server components - let's use it, you will find more details here: https://nextjs.org/docs/app/building-your-application/routing